awswrangler/sqlserver.py

```diff
@@ -291,6 +291,7 @@ def to_sql(
     dtype: Optional[Dict[str, str]] = None,
     varchar_lengths: Optional[Dict[str, int]] = None,
     use_column_names: bool = False,
+    chunksize: int = 200,
```

Comment:
Use the config class to set this value and make available to users or other modules to change this in code.

Reply:
Done 👍